features/steps/command/shell.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Running shell commands
+
+  You can run an *ad hoc* script with the following step
+  - `When I run the following script:`
+
+  Or you can run shell commands with:
+  - `I run the following (bash|zsh|fish|dash)? commands
+
+  Background:
+    Given I use a fixture named "cli-app"
+
+  Scenario: Running script
+    Given a file named "features/shell.feature" with:
+    """
+    Feature: Running scripts
+      Scenario: Running ruby script
+        When I run the following script:
```

Comment:
Seems like a good idea to get ride of the `name` this way!!

About merging the two step i'm not sure. Having just to copy the script there without modifying 
the shebang would be better, and more clear on read

(what code section are you targeting?)